src/plugins/intel_gpu/src/plugin/transformations/disable_fp16_comp_rms.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "disable_fp16_comp_rms.hpp"
+
+#include "ov_ops/rms.hpp"
+#include "openvino/op/add.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/core/rt_info.hpp"
+#include "openvino/pass/pattern/op/wrap_type.hpp"
+#include "transformations/utils/utils.hpp"
+
+#include <memory>
+
+namespace ov::intel_gpu {
```

Comment:
could you point me to the place where clamp is added explicitly? I expected your new impl to replace the old impl(adding clamp).. but I think I got misunderstanding.